pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -108,16 +115,23 @@ func validateFilters(filters []string, fieldName string) (causes []metav1.Status
 		return nil
 	}
 
+	filedString := k8sfield.NewPath("spec").Child(fieldName).String()
+	errPattern := "%s filter %s is invalid: cannot contain a %s character (%s); FilterRules: %s"
+	if s, substr := strings.ToUpper(fieldName), strings.ToUpper("template"); strings.Contains(s, substr) {
```

Comment:
Control this with a bool instead of string parsing? `templateField bool`?

Reply:
OK，I can't agree any more.